server/container/guice/jmx/src/main/java/org/apache/james/modules/server/JmxConfiguration.java

```diff
@@ -34,6 +34,11 @@ public class JmxConfiguration {
     public static final String LOCALHOST = "localhost";
     public static final int DEFAULT_PORT = 9999;
     public static final boolean ENABLED = true;
+    public static final String JMX_CREDENTIAL_GENERATION_ENABLE_PROPERTY = "james.jmx.credential.generation";
+    public static final String JMX_CREDENTIAL_GENERATION_ENABLE_DEFAULT = "false";
+    public static final String PASSWORD_FILE_PATH = "/root/conf/jmxremote.password";
+    public static final String ACCESS_FILE_PATH = "/root/conf/jmxremote.access";
```

Comment:
This location is the docker one but we should not use that for raw installation. 

Suggestion:

```
org.apache.james.filesystem.api.FileSYstem.getFile("conf://jmxremote.password");
```